src/lib.rs

```diff
@@ -11,19 +11,19 @@
 //! * `GET /api/info` returns an informative [`api::InfoResponse`] object.
 //! * `POST /api/run` runs a command analogous to [`std::process::Command`].
 //! * `POST /api/runscript` runs a script with a given interpreter.
-//! * `GET /api/file` fetches a file from the servers file system.
+//! * `GET /api/file/{path}` fetches a file from the servers file system.
 //!
 //! ## Working with files
 //! The working directory of the executed commands is implementation defined,
 //! but the same for all methods and constant over the lifetime of the server.
-//! The path for file fetching is also relative to this directory.
+//! The path for file fetching is also relative to (and confined to!) this directory.
```

Comment:
```suggestion
//! The path for file fetching is also a relative path in this directory.
```

Reply:
I'd rather be explicit in that the path is confined to that directory.

Reply:
Yeah, but the exclaimation mark and parathesis are too strong of a statement. Make it "a relative path inside this directory", then it is even clearer without making the wrong point. The point isn't that you shouldn't be able to access anything else, hell this is RCEaaS, it is that you know wether to give a full path or a just the relative part.